packages/vite/src/node/ssr/index.ts

```diff
@@ -35,14 +35,14 @@ export interface SSROptions {
      *
      * @default rootConfig.resolve.conditions
      */
-    conditions?: string[]
+    conditions?: readonly string[]
```

Comment:
Wouldn't marking this `readonly` make it not possible to `.push()` in `config` or `configEnvironment`?

Reply:
That's true. I'll try to solve this in a different way.

Reply:
Fixed by bda9b754fe84c6b014860640fcb9e7f8fc846797 👍